Core/HLE/sceKernelSemaphore.cpp

```diff
@@ -457,3 +458,26 @@ int sceKernelPollSema(SceUID id, int wantedCount)
 	}
 }
 
+static u32 hleUtilsBufferCopyWithRange(u32 outAddr, int outSize, u32 inAddr, int inSize, int cmd)
+{
+		sceUtilsBufferCopyWithRange((u8*)outAddr, outSize, (u8*)inAddr, inSize, cmd);
+		// Fake a successful operation
+		return 0;	
```

Comment:
Please log something when it goes wrong even if it helps to fake a successful operation here.

(also, the indentation is off by one).